libtransmission/torrent.cc

```diff
@@ -148,16 +148,22 @@ bool tr_torrentSetMetainfoFromFile(tr_torrent* tor, tr_torrent_metainfo const* m
 
 namespace
 {
-bool setLocalErrorIfFilesDisappeared(tr_torrent* tor, std::optional<bool> has_local_data = {})
+bool if_files_disappeared(tr_torrent* tor, std::optional<bool> has_local_data = {})
```

Comment:
> ```if_files_disappeared```

Small typo: ```if_``` --> ```is_```.

Reply:
Hmm, the phrase "is files disappeared" doesn't make sense to me... If you are not happy with "if files disappeared", is "did files disappear" better for you?

Reply:
> I am guessing you are just following the practice ...

Right. But if this practice is not demanded, then the naming is fully upon you. (Anyway your suggestion sounds well to me. Since the function is used in "if" statement it looks definitively better if the name wouldn't start with "if".)

Reply:
098c5cf